controlpanel/frontend/jinja2/base.html

```diff
@@ -107,16 +108,16 @@
         "active": page_name == "webapps"
       },
       {
-        "hide": not request.user.is_superuser,
         "text": "Parameters",
         "href": url("list-parameters"),
-        "active": page_name == "parameters"
+        "active": page_name == "parameters",
```

Comment:
maybe add the trailing comma to L120 instead?

Reply:
:slightly_smiling_face: so this isn't consistent with the rest of the dicts defined in the file. I personally prefer to add commas, but whoever did this work was obviously of the "other school". ;-)

Reply:
@ntoll or they may have just forgot 😬 - in the future feel free to add trailing commas in part of the code you're touching - unless of course is syntactically wrong (I'm looking at you JSON! 😬)

I don't think anyone would tell you off for being inconsistent for adding a trailing comma 😉

Reply:
OK... with the latest commit I've taken the executive decision to add training commas to all the appropriate things in the `base.html` file.

For discussion and perhaps a further PR, what do folks think about using [Black](https://black.readthedocs.io/en/stable/) to automagically format our code for us so we don't need any arbiter of "code style"? As the Black docs say, "you can have any colour so long as..." FWIW after an initial "wat?" reaction I've had fantastic experience of using Black and would recommend it without hesitation.